media-libs/panda3d/panda3d-9999.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 2008-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit eutils multiprocessing python-single-r1
+
+if [[ ${PV} != 9999 ]]; then
+	MY_PV=v${PV}
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="http://www.panda3d.org/download/${P}/${P}.tar.gz"
+	S="${WORKDIR}"/${P}
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/panda3d/panda3d.git"
+	EGIT_BRANCH="master"
+fi
+
+DESCRIPTION="A 3D game engine and framework for Python and C++."
+HOMEPAGE="http://www.panda3d.org"
+
+LICENSE="BSD"
+SLOT="0"
+
+IUSE="bullet +eigen ffmpeg fftw fmod jpeg opencv png +python +openal +ssl tiff truetype zlib"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	bullet? ( sci-physics/bullet )
+	eigen? ( dev-cpp/eigen:3 )
+	ffmpeg? ( media-video/ffmpeg )
+	fmod? ( media-libs/fmod:= )
+	jpeg? ( virtual/jpeg:= )
+	opencv? ( media-libs/opencv )
+	png? ( media-libs/libpng:= )
+	${PYTHON_DEPS}
+	openal? ( media-libs/openal )
+	ssl? ( dev-libs/openssl:= )
+	tiff? ( media-libs/tiff:= )
+	truetype? ( media-libs/freetype )
+	zlib? ( sys-libs/zlib )
+	virtual/opengl"
+RDEPEND="${DEPEND}"
+# Note: libav may or may not work instead of ffmpeg, but in any case is not recommended by upstream and untested. If you try and it works please add it as an option, but include a warning in postinst.
+
+src_compile() {
+	multijob_init # mandatory according to docs
+	python_setup # unconditionally calling this as we need python for building it, even with USE -python
+	"${EPYTHON}" ./makepanda/makepanda.py \
+		--everything \
+		--$(usex bullet use)-bullet \
+		--$(usex eigen use)-eigen \
+		--$(usex ffmpeg use)-ffmpeg \
+		--$(usex fftw use)-fftw \
+		--$(usex fmod use)-fmod \
+		--$(usex jpeg use)-jpeg \
+		--$(usex png use)-png \
+		--$(usex opencv use)-opencv \
+		--$(usex python use)-python \
+		--$(usex openal use)-openal \
+		--$(usex ssl use)-openssl \
+		--$(usex tiff use)-tiff \
+		--$(usex truetype use)-freetype \
+		--$(usex zlib use)-zlib \
+		--no-fcollada \
+		--no-gles \
+		--no-gles2 \
+		--no-ode \
+		--no-rocket \
+		--no-squish \
+		--no-vrpn \
+		--threads $(makeopts_jobs) \
+		|| die "build failed"
+
+		# note: The X11/OpenGL stuff is optional, so a USE flag for that could be added. For such builds drop the --everything
+}
+
+src_install() {
+	"${EPYTHON}" ./makepanda/installpanda.py --destdir "${D}" --prefix "/usr" || die "install failed"
+}
+
+pkg_postinst()
+{
+	elog
+	# TODO fix and reactivate doc
+	# had "doc? ( dev-python/epydoc )" in dependencies but that wasn't used anywhere
+	#if use doc ; then
+	#	elog "Documentation is installed in /opt/panda3d/doc"
+	#	elog "Samples are installed in /opt/panda3d/samples"
+	#fi
+	elog "For C++ compiling, include directory must be set:"
+	elog "g++ -I/opt/panda3d/include [other flags]"
```

Comment:
oops. the old ebuild did. since we now use standard paths i'm just dropping the comment.